airbyte-integrations/connectors/destination-pinecone/integration_tests/pinecone_integration_test.py

```diff
@@ -48,6 +50,8 @@ def tearDown(self):
             else :
                 print("Noting to delete. No data in the index/namespace.")
 
+    def test_integration_test_flag_is_set(self):
+        assert os.getenv("INTEGRATION_TEST", False) == "True"
```

Comment:
```suggestion
        assert "PYTEST_CURRENT_TEST" in os.environ
```
 
TIL 🙂

https://stackoverflow.com/a/58866220/4298208